repose-aggregator/commons/commons-utilities/src/main/scala/org/openrepose/commons/utils/servlet/http/HttpServletResponseWrapper.scala

```diff
@@ -75,7 +76,8 @@ import scala.io.Source
   *                            depending on mode selection of the body
   */
 class HttpServletResponseWrapper(originalResponse: HttpServletResponse, headerMode: ResponseMode, bodyMode: ResponseMode, desiredOutputStream: ServletOutputStream)
-  extends javax.servlet.http.HttpServletResponseWrapper(originalResponse) with HeaderInteractor {
+  extends javax.servlet.http.HttpServletResponseWrapper(originalResponse) with HeaderInteractor with LazyLogging {
```

Comment:
I think we've settled into putting each of these on their own line with the `with`s being further tabbed in.

Reply:
For whatever reason, I never got in the habit of doing that, but it makes sense. Oh, and the Scala style guide agrees:
https://docs.scala-lang.org/style/declarations.html#classes

Will update.